algebra/src/serialize/mod.rs

```diff
@@ -144,6 +155,34 @@ macro_rules! impl_sw_curve_serializer {
                 Ok(p)
             }
         }
+
+        impl<P: $params> GroupSerialize for GroupAffine<P> {
+            fn serialize_uncompressed(
+                &self,
+                output_buf: &mut [u8],
+            ) -> Result<(), crate::serialize::SerializationError> {
+                let len = <Self as CanonicalSerialize>::buffer_size();
```

Comment:
What about using the field buffer size here instead?

Reply:
Good idea, fixed: https://github.com/scipr-lab/zexe/pull/99/commits/3039fc1d5885753ea4317914744b111d614d2afc#diff-40da71ffac4550ada607d2de65a09100R164